src/compas_viewer/scene/polyhedronobject.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+from compas.datastructures import Mesh
+from compas.geometry import Line
+from compas.geometry import Point
+from compas.geometry import Polyhedron
+from compas.scene import GeometryObject
+
+from .geometryobject import GeometryObject as ViewerGeometryObject
+
+
+class PolyhedronObject(ViewerGeometryObject, GeometryObject):
+    """Viewer scene object for displaying COMPAS Polyhedron geometry.
+
+    See Also
+    --------
+    :class:`compas.geometry.Polyhedron`
+    """
+
+    def __init__(self, polyhedron: Polyhedron, **kwargs):
+        super().__init__(geometry=polyhedron, **kwargs)
+        self.geometry: Polyhedron
+        self.show_points = False
+        self.show_lines = True
+
+    @property
+    def points(self) -> Optional[list[Point]]:
+        """The points to be shown in the viewer."""
+        return self.geometry.points
+
+    @property
+    def lines(self) -> Optional[list[Line]]:
+        """The lines to be shown in the viewer."""
+        return self.geometry.lines
+
+    @property
+    def viewmesh(self):
+        """The mesh volume to be shown in the viewer."""
+        mesh = self.geometry.to_mesh()
+        vert, face = mesh.to_vertices_and_faces(triangulated=True)
+        mesh = Mesh.from_vertices_and_faces(vert, face)
+        return mesh
```

Comment:
how was this done in the old viewer?

Reply:
> Should we just say faces?

yes sure

Reply:
how is it done for polygons? every face is basically a polygon?

Reply:
On the side, if we choose to not do anything about above points. The way we can make it cleaner is to do the same like other geometry objects:
https://github.com/compas-dev/compas_viewer/blob/32ea97a80dcaa4371e68d0017bebd544836db19c/src/compas_viewer/scene/boxobject.py#L53

For this to work, the `to_vertices_and_faces` function of Polyhedron needs to accept triangulated argument. Which I can quickly make a PR on COMPAS core.

Reply:
i think i misunderstood your remark about `viewmesh`. i though you meant that this was a bad name in general, not just for the `Polyhedron`. i would prefer the name to be consistent with other scene objects. in any case, `volume` makes no sense since a polyhedron is not necessarily closed. it is just a collection of vertices and faces...